NEWS.md

```diff
@@ -1,3 +1,28 @@
+## synapser 2.1.0
+
+### Improvements
+
+* Upgraded to the synapsePythonClient v4.4.0.
+* The data upload and download algorithm in synapsePythonClient have been revamped for enhanced stability, reliability, and performance.
+* Updated reference documents and code examples for both new and modified functions. 
+
+#### New Functions
+* `synCreateTeam` and `synDeleteTeam` have been added to manage team. 
+* `synRestGetAsync`, `synRestDeleteAsync`, `synRestPostAsync` and `synRestPutAsync` have been added to allow interaction with Synapse server utilizing [asynchronous](https://python-docs.synapse.org/reference/oop/models_async/) models.
+
+#### Minor behavior changes: 
+* Credentials passed by command line argument will now be evaluated before credentials stored in the `~/.synapseConfig` file.
+* `rememberMe` has been deprecated in `synLogin`. 
```

Comment:
Nope this is a change made in 4.4.0.

Reply:
Can you confirm that? I thought it was done in 4.0.0: https://python-docs.synapse.org/news/#400-2024-01-12

Reply:
Sorry. Do you refer to the second bullet point instead of first one? This deprecation didn't specify in the changelog and was still in the [reference docs](https://github.com/Sage-Bionetworks/synapser/blob/2.0.0/man/synLogin.Rd) for v2.0.0. So I added here.

Reply:
Thanks @danlu1, I see. I wonder if we should add it to the 2.0.0 section so it's not confusing since these have technically been true since the last release.

The bullet point in talking about is the deprecate rememberMe

Reply:
Sure. Will do.